src/main/java/mate/academy/spring/controller/UserController.java

```diff
@@ -0,0 +1,46 @@
+package mate.academy.spring.controller;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import mate.academy.spring.model.User;
+import mate.academy.spring.model.dto.UserResponseDto;
+import mate.academy.spring.service.UserService;
+import mate.academy.spring.service.mapper.UserDtoMapper;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
```

Comment:
```suggestion
@RestController("/users")
```
можно сделать общую ссылку для класса